os/fs/smartfs/smartfs.h

```diff
@@ -221,8 +221,9 @@
 
 /* Buffer flags (when CRC enabled) */
 
-#define SMARTFS_BFLAG_DIRTY       0x01	/* Set if data changed in the sector */
-#define SMARTFS_BFLAG_NEWALLOC    0x02	/* Set if sector not written since alloc */
+#define SMARTFS_BFLAG_DIRTY		0x01	/* Set if data changed in the sector */
+#define SMARTFS_BFLAG_NEWALLOC		0x02	/* Set if sector not written since alloc */
```

Comment:
1. Code align is needed here.
2. To use this values with bit operation, 0x04 is correct for new one.

Reply:
Updated